pkg/agent/flowexporter/connections/interface.go

```diff
@@ -27,3 +29,9 @@ type ConnTrackDumper interface {
 	// GetMaxConnections returns the size of the connection tracking table.
 	GetMaxConnections() (int, error)
 }
+
+// ConnectionStoreInterface is used to support exporting expired connections from
+// conntrack connection store and deny connection store.
+type ConnectionStoreInterface interface {
+	GetExpiredConns(currTime time.Time) ([]flowexporter.Connection, time.Duration)
```

Comment:
As Conntrack and Deny connection stores have slightly different implementations, interface with connection store is not useful here. If they have the same implementation, then the interface would be useful.
We could make it the same but you will need an extra argument to know if the connection store is conn track or deny or have a field in the connection to determine if it's conntrack or deny. I prefer the first one, but this method breaks the common code principle in connection store.

Instead of doing above,I am ok with a separate implementation.

Reply:
Thanks Srikar. I was also confusing about what's the benefit for having an interface here. Removed the interface, kept a separate implementation here.